src/actions/authentication.js

```diff
@@ -23,7 +25,10 @@ export const signIn = async (payload) => {
     await Api.signIn(payload);
     notifier.success({ code: SIGN_IN_SUCCESS });
   } catch (error) {
-    notifier.error({ code: SIGN_IN_ERROR, error });
+    if(error.message === 'Not Found')
```

Comment:
Should we create an `enum` for these error message strings such as `'Not Found'`?

Reply:
I don't like how this is done, if the request doesn't return a success code (200-299) it throws an error with the response message.

I think it would be better to not throw an error and the caller needs to check the status of the request.  

Some thing like this would be cleaner :
```
const res = await Api.signIn(payload);
if(res.ok){
    notifier.success({ code: SIGN_IN_SUCCESS });
}
else{
   switch(res.status){
   ...  
  }
}
```

Any thoughts on this @juancarlosfarah @pyphilia

Reply:
Added response in error and used [http-status-codes ](https://www.npmjs.com/package/http-status-codes ) to compare with code instead of message